_layouts/post.html

```diff
@@ -21,7 +21,7 @@ <h1 data-toc-skip>{{ page.title }}</h1>
     </span>
 
     <!-- lastmod date -->
-    {% if page.last_modified_at %}
+    {% if page.last_modified_at != page.date %}
```

Comment:
There is a bug in this condition. If a post has not modified, an unexpected "update time" will be generated. Therefore, the condition should be improved:

```suggestion
    {% if page.last_modified_at != page.date and page.last_modified_at %}
```